sw/airborne/firmwares/rotorcraft/autopilot_static.c

```diff
@@ -173,8 +173,14 @@ void autopilot_static_periodic(void)
     SetCommands(commands_failsafe);
   } else {
     guidance_v_run(autopilot_in_flight);
-    guidance_h_run(autopilot_in_flight);
+    guidance_h_run(autopilot_in_flight);    
     SetRotorcraftCommands(stabilization_cmd, autopilot_in_flight, autopilot_motors_on);
+
+    if (lock_wings == 1 && autopilot_motors_on && commands[COMMAND_THRUST] > 0) {
```

Comment:
It is true that in your case the AP/FBW split is annoying. Then I'm suggesting to make the `SetRotorcraftCommand` configurable. Currently there is 3 per-defined case, but we could put a `#ifndef SetRotorcraftCommand` around that (and include `modules.h` before) so that you can define your own set command function in a module. What do you think ?